### DISCUSSION TITLE
How to document tuple

### DISCUSSION THREAD

**Discussion by tomhelmer** [dedoc/scramble#213] [Q&A]
Hi, I'm documenting an API with GeoJSON where 'coordinates' is a float tuple. 

See example on the frontpage: https://geojson.org/

The closest I can get is using a docblock comment :

```
'geometry' => [
                'type' => 'Point',
                /** @var float[] */
                'coordinates' => [$this->lng, $this->lat]
            ],
```

But it results in:

```
"geometry": {
    "type": "Point",
    "coordinates": [
      0
    ]
  },
```

Is there a way to manually add the example data ? ( or some other means of solving this )

**Comment by romalytvynenko** [comment]
Hey @tomhelmer 

This is a great question. For now there is no easy support of tuples in OpenAPI spec. There are some workarounds, but Scramble doesn't use that for now.

I have created an issue for this matter. I will check how tuples can be documented, and also I'll add support of examples to response schemas (currently examples are supported only for request parameters).

You can keep an eye on the issue: https://github.com/dedoc/scramble/issues/218

Thanks!
